core/server/models/base/index.js

```diff
@@ -114,7 +114,7 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
             newObj.setId();
         }
 
-        if (!this.get('created_by')) {
+        if (schema.tables[this.tableName].hasOwnProperty('created_by') && !this.get('created_by')) {
```

Comment:
This is 👍  but maybe needs a comment or a util/helper function to make what it is doing clearer outside of the context of this commit?